### PR TITLE
tweaks to equations

### PR DESCRIPTION
**Pull request by simonbyrne**
Based on suggestions from @tapios

### CODE REVIEW

File: docs/src/equations.md [hunk 1]
```diff
@@ -31,31 +33,33 @@ We make use of the following operators
...
   ```math
   \boldsymbol{\Omega} = \Omega \sin(\phi) \boldsymbol{e}^v
   ```
-  where ``\phi`` is latitude, and ``\Omega = 7.29212 \times 10^{-5}`` is the rotation rate (in rads/sec) and ``\boldsymbol{e}^v`` is the normal radial basis vector. This implies that the horizontal contravariant component ``\boldsymbol{\Omega}^h`` is zero.
-* ``\Phi = g z`` is the geopotential, where ``g`` is the gravitational acceleration rate and ``z`` is altitude.
-* ``K = \tfrac{1}{2} \|\boldsymbol{u}\|^2 `` is the kinetic energy per mass (J/kg), reconstructed at cell centers by
-```math
-K = \tfrac{1}{2} \|\boldsymbol{u}_h + I^c(\boldsymbol{u}_v)\|^2 .
-```
+  where ``\phi`` is latitude, and ``\Omega`` is the planetary rotation rate in rads/sec (typically 7.29212 \times 10^{-5}) and ``\boldsymbol{e}^v`` is the unit radial basis vector. This implies that the horizontal contravariant component ``\boldsymbol{\Omega}^h`` is zero.
```
Comment (tapios) [inline docs/src/equations.md:56]:
```suggestion
  where ``\phi`` is latitude, and ``\Omega`` is the planetary rotation rate in rads/sec (for Earth, 7.29212 \times 10^{-5}) and ``\boldsymbol{e}^v`` is the unit radial basis vector. This implies that the horizontal contravariant component ``\boldsymbol{\Omega}^h`` is zero.
```

**Comment by valeriabarra** [comment]
Hi @simonbyrne , I wonder if you had seen #336  where I had tagged you as a reviewer. I had already tweaked a couple of things there

File: docs/src/equations.md [hunk 2]
```diff
@@ -31,31 +33,33 @@ We make use of the following operators
...
+* ``\Phi = g z`` is the geopotential, where ``g`` is the gravitational acceleration rate and ``z`` is altitude above the mean sea level.
+* ``K = \tfrac{1}{2} \|\boldsymbol{u}\|^2 `` is the specific kinetic energy (J/kg), reconstructed at cell centers by
+  ```math
+  K = \tfrac{1}{2} \|\boldsymbol{u}_h + I^c(\boldsymbol{u}_v)\|^2.
+  ```
+  where ``\|\boldsymbol{u}\|^2 = g^{ij} \boldsymbol{u}_i \boldsymbol{u}_j`` and ``g^{ij}`` is the metric tensor.
```
Comment (tapios) [inline docs/src/equations.md:62]:
```suggestion
  where ``\|\boldsymbol{u}\|^2 = g^{ij} \boldsymbol{u}_i \boldsymbol{u}_j`` and ``g^{ij}`` is the contravariant metric tensor.
```

**Pull request merged**